hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -206,29 +213,57 @@ def lookup_get():
      Also (applies to both cases):
      * Optional seed (content id) for consistent coinflip
     Output:
-     * JSON object with a key of each bank name with matches.
-       For each bank, there is list of objects containing details
-       about each match.
+     * JSON object keyed by signal type, to a JSON object of bank
+       matches. If Signal Type is given, the outer JSON object is
+       elided.
+
+    Example output:
+    {
+        "pdq": {
+            "BANK_A": [
+                {"bank_content_id": 1001, "distance": 4},
+                {"bank_content_id": 1002, "distance": 0},
+            ],
+            "BANK_B": [
```

Comment:
I personally find this nesting a bit strange, I'd prefer banks be on the "outside".